source/client-side-encryption/tests/README.rst

```diff
@@ -2315,3 +2315,67 @@ Set ``X-MongoDB-HTTP-TestParams`` to ``case=slow``.
 
 The HTTP response from the ``fake_azure`` server will take at least 1000 seconds
 to complete. The request should fail with a timeout.
+
+19. Azure IMDS Credentials Integration Test
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Refer: `Automatic Azure Credentials <auto-azure_>`_
+
+.. _auto-azure: ../client-side-encryption.rst#obtaining-an-access-token-for-azure-key-vault
+
+For these cases, create a ClientEncryption_ object :math:`C` with the following
+options:
+
+.. code-block:: typescript
+
+   ClientEncryptionOpts {
+      keyVaultClient: <setupClient>,
+      keyVaultNamespace: "keyvault.datakeys",
+      kmsProviders: { "azure": {} },
+   }
+
+Case 1: Failure
+```````````````
+
+Do not run this test case in an Azure environment with an attached identity.
+This may be run in an AWS EC2 instance.
```

Comment:
can this be just a regular env without any configuration? If so, we can get rid of special EG configuration for this case